### ISSUE TITLE
Questions for the inveter

### ISSUE THREAD

**Issue by JensPauwels** [Emilv2/huawei_solar#63]
1) Is there a reason for the negative watt peak from the inverter?

![image](https://user-images.githubusercontent.com/17976979/146672563-8bbbbbb4-0645-4005-ad74-cdedd4aa3a55.png)

2) There's a weird behaviour in the daily_yield. I'm currently using this energy grid setup:
![image](https://user-images.githubusercontent.com/17976979/146672654-590c4628-1ae8-44af-bd5c-946d7ff3c175.png)

However the daily_yield also shows some values at night, which is find very weird. Is it possible to explain how the daily yield works, because perhaps i'm misunderstanding it

![image](https://user-images.githubusercontent.com/17976979/146672639-f548c88f-21de-47c6-a554-7090950b7e23.png)

**Comment by Emilv2** [comment]
No, that's a bug. You better use total yield for the energy panel.

**Comment by JensPauwels** [comment]
Thank you, the total yield indeed looks more realistic. Is there an explanation for the night yields?

**Comment by Emilv2** [comment]
The negative yield around 00:00 is because the reset time is not correctly synced, I thought I coded this correctly but apparently not. I don't really understand what's going on with the night yields or the negative peak.

**Comment by JensPauwels** [comment]
The host is a debian server which is synced, is it a setting in home assistant that I need to change to prevent this or will you fix this in a next version?
